pyproject.toml

```diff
@@ -1,3 +1,7 @@
 [build-system]
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[tool.ruff]
+line-length = 88
```

Comment:
isn't this the default?

Reply:
Yeah, it is -- no harm in specifying it though right?